app/viewtils/PenaltyTypeHelper.scala

```diff
@@ -30,7 +30,7 @@ object PenaltyTypeHelper {
     penaltyAsEnum match {
       case Some(PenaltyTypeEnum.Late_Submission) => Some(messages("penaltyType.lateSubmission"))
       case Some(PenaltyTypeEnum.Late_Payment) => Some(messages("penaltyType.latePayment"))
-      case Some(PenaltyTypeEnum.Additional) => Some(messages("penaltyType.additional"))
+      case Some(PenaltyTypeEnum.Additional) => Some(messages("penaltyType.latePayment"))
```

Comment:
This needs changing to `penaltyType.additional`
```suggestion
      case Some(PenaltyTypeEnum.Additional) => Some(messages("penaltyType.additional"))
```

Reply:
The value used against penaltyType.latePayment  ie Late payment penalty is shown in the AppealConfirmationPage.